modules/dnn/test/test_onnx_conformance.cpp

```diff
@@ -1222,6 +1222,12 @@ TEST_P(Test_ONNX_conformance, Layer_Test)
         {
             if (ref_outputs.size() == 1)
             {
+                if (ref_outputs[0].dims != outputs[0].dims) {
+                    if (ref_outputs[0].dims <= 1)
+                        ref_outputs[0] = ref_outputs[0].reshape(1, outputs[0].rows);
+                    if (outputs[0].dims <= 1)
+                        outputs[0] = outputs[0].reshape(1, ref_outputs[0].rows);
+                }
```

Comment:
What is the shape of `ref_outputs[0]` and `outputs[0]` in this case? It looks weird to me that they need to be handled additionally.

Reply:
`ref_outputs[0] = 1x1`
`output[0] = 1`